server/services/reportService.ts

```diff
@@ -63,6 +73,20 @@ export default class ReportService {
     return report
   }
 
+  async getReportInProgress(bookingId: number): Promise<inProgressReport | null> {
+    const report = await this.incidentClient.getReportInProgress(bookingId, ReportStatus.IN_PROGRESS.value)
+    return report
+  }
+
+  async getReportsByDate(bookingId: number, incidentDate: string): Promise<offenderReports[] | []> {
+    const startDate = moment(incidentDate, 'DDMMYYYY')
+    const endDate = moment(incidentDate, 'DDMMYYYY').add('1', 'd')
+    const reports = await this.incidentClient.getReportsForAnOffenderForSpecificDate(bookingId, [startDate, endDate])
+    return reports
+      .filter(report => report.status !== ReportStatus.IN_PROGRESS.value)
```

Comment:
strip out any in progress reports as the intention is to see if there are already submitted/completed reports only